example-config.yml

```diff
@@ -115,6 +115,7 @@ persistence:
 map:
   initLat: 45.52
   initLon: -122.682
+  # autoFlyOnTripFormUpdate: false
```

Comment:
If we could add a description of the default behavior here too, depending on what we land on